packages/lime-cc/Makefile

```diff
@@ -0,0 +1,60 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=lime-cc
+PKG_VERSION:=0.0.2
+PKG_RELEASE:=2
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/lime-cc-client
```

Comment:
As you're the naming guru, you have a suggestion? `lime-cc-bmx7`? It could be split in different modules, one for `bmx7`, another for `bmx6` + `usign` signature, one for server polling?

Reply:
I'd set it to something like lime-bmx7-remote-controller Since the conept "cloud" applies to link-local domains, so bat-adv.